test/test_places.js

```diff
@@ -32,6 +32,22 @@ describe('Places', () => {
 
         }).catch(done)
 
+    }),
+
+    it('Search by IDs', (done) => {
+
+    let c = new Client({ access_token : test_user_password_access_token})
+
+    c.places.search({id : ['5128638']})
```

Comment:
Would you mind adding another test to make sure fetching 2 places works as expected? Fetching 1 place was fine, but multiple was failing.